llvm/test/CodeGen/SPIRV/structurizer/basic-if.ll

```diff
@@ -0,0 +1,53 @@
+; RUN: llc -mtriple=spirv-unknown-vulkan-compute -O0 %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-vulkan-compute %s -o - -filetype=obj | spirv-val %}
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-G1"
```

Comment:
Why explicit datalayout?

Reply:
Just took that from the LLVM output. Removed.